src/app/GitUI/CommandsDialogs/FormRenameBranch.cs

```diff
@@ -51,22 +51,10 @@ private void OkClick(object sender, EventArgs e)
                 return;
             }
 
-            try
-            {
-                string renameBranchResult = Module.RenameBranch(_oldName, newName);
```

Comment:
I would have preferred to use the return status from the command instead, and present the error text.
You hate error hiding, I dislike popups that are not needed
Not requiring a change

Reply:
> I dislike popups that are not needed

The popup is transient if "Keep dialog open" is unticked and no error occurred.
(AD: The success output can be read in the Output History if the popup closes too fast. I.e. there is no need to "Keep dialog open" any longer. [FormProcess could even be merged into the Output History, i.e. less popups.])

Reply:
>  (AD: The success output can be read in the Output History if the popup closes too fast. I.e. there is no need to "Keep dialog open" any longer. [FormProcess could even be merged into the Output History, i.e. less popups.])

Sidenote:
that is #10307 that I have approved and support, have used it even if I not have used it much. 
I mostly use the popup to open/view PRs from BitBucket/GitHub urls and that is more easily available and I have not changed how I work.
It has not been high on my agenda though to convince RussKie there.

So also this could be better if I change the way I work.
Right now I am mostly neutral, not enthusiastic at this moment... (Take this literally, no understatement.)